components/Competition/GameDetail/index.js

```diff
@@ -16,7 +16,7 @@ import {
 
 export default function GameDetail({
   game,
-  gameNumber,
+  gamenumber,
```

Comment:
This should be camel case. Why did you change it?

Reply:
There was one warning coming from react on a `Link` where I was passing it as camel-case and I wanted to change it just there but it seems I changed it on all places accidentally. I will revert it and just fix it on this place 🙂
Thanks for the hint!